### PR TITLE
Fix Site model import

### PR DESCRIPTION
**Pull request by danielsousaio**
UPDATE:
For future readers, this PR adds a missing historical dependency on this data migration. Without it, we can't guarantee that the sites table is created. Further, get_model() requires the historical version of the model granted by adding the dependency.

### CODE REVIEW

File: {{cookiecutter.project_slug}}/home/migrations/0002_load_initial_data.py [hunk 1]
```diff
@@ -21,7 +22,6 @@ def create_homepage(apps, schema_editor):
 
 
 def create_site(apps, schema_editor):
-    Site = apps.get_model("sites", "Site")
```
Comment (driverdan) [inline {{cookiecutter.project_slug}}/home/migrations/0002_load_initial_data.py]: This needs to stay

Comment (danielsousaio) [reply {{cookiecutter.project_slug}}/home/migrations/0002_load_initial_data.py]: Import at the top, this one doesn't work, tested on staging.

Comment (driverdan) [reply {{cookiecutter.project_slug}}/home/migrations/0002_load_initial_data.py]: It should work now that it depends on a sites migration. See the docs here: https://docs.djangoproject.com/en/3.0/topics/migrations/#accessing-models-from-other-apps

**Comment by danielsousaio** [comment]
For future readers, this PR adds a missing historical dependency on this data migration. Without it, we can't guarantee that the sites table is created. Further, `get_model()` requires the historical version of the model granted by adding the dependency.

**Pull request merged**